docs/simon.rst

```diff
@@ -63,12 +63,12 @@ whereas the best classical algorithm requires exponential time.
 Source Code Docs
 ----------------
 
-Here you can find documentation for the different submodules in phaseestimation.
+Here you can find documentation for the different submodules in simon.
 
 grove.simon.simon
```

Comment:
Why does this have to be `grove.simon.simon` instead of `grove.simon`?

Reply:
@marcusps I *think* it's because the path here is meant to reference a module, and the package structure is such that `simon.py` lives in the `simon` folder. This is the case with `qft`. But I'm currently struggling to get the docs to generate the source code in this example.

Reply:
Oh, nevermind, I see what you mean. I don't think this label does anything, I think the one below is the one that matters.

Reply:
@marcusps: @ampolloreno is correct. The current module structure makes this necessary. There is an additional layer of imports that will be removed later on.